invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/Document/DocumentStats.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { Icon, Table, Message } from 'semantic-ui-react';
+import { Error } from '../../../../common/components';
+import { http } from '../../../../common/api/base';
+import { FrontSiteRoutes } from '../../../../routes/urls';
+
+export class DocumentStats extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: { views: { count: '-', unique_count: '-' } },
+      error: {},
+    };
+  }
+
+  componentDidMount() {
+    this.fetchStats();
+  }
+
+  fetchStats = async () => {
+    const { document } = this.props;
+    const request = {
+      views: {
+        stat: 'record-view',
+        params: {
+          pid_value: document.pid,
+        },
+      },
+    };
+    try {
+      const response = await http.post(FrontSiteRoutes.statsUrl, request);
```

Comment:
Just reading this, I think it should be: `documentApis.statsUrl` no?

Reply:
I wasn't really sure where to put it, thanks!